open-reac/src/test/java/com/powsybl/openreac/OpenReacParametersTest.java

```diff
@@ -39,6 +40,52 @@ public void testObjectiveIntegrityChecks() {
         assertDoesNotThrow(() -> parameters.checkIntegrity(network), "Default configuration with BETWEEN_HIGH_AND_LOW_VOLTAGE_LIMIT and ratio voltage set should not throw");
     }
 
+    @Test
+    void testMinMaxVoltageLimitIntegrityChecks() {
```

Comment:
Can you provide a unit test that export the file with parameters and compare to a reference file stored in the ressources?